patches/chromium/feat_enable_passing_exit_code_on_service_process_crash.patch

```diff
@@ -32,86 +36,86 @@ index e900841900325dd090b04f85151dc7b2f04a6613..e402bcf07aa0704c113ce472db547ffc
    processes_.erase(iter);
  }
 diff --git a/content/browser/service_host/service_process_tracker.h b/content/browser/service_host/service_process_tracker.h
-index 8e2b330f59c714db9dac3363df566b6e396fa4ec..47b0b3a944dec09a229fe771813118bbe49b8e28 100644
+index 9a5179c4eeacf8bbfb2d831b4301836df490f3a8..511fc9b9d5ce14a1b5ef457a1047e40d3bb64273 100644
 --- a/content/browser/service_host/service_process_tracker.h
 +++ b/content/browser/service_host/service_process_tracker.h
-@@ -34,7 +34,7 @@ class ServiceProcessTracker {
- 
+@@ -36,7 +36,8 @@ class ServiceProcessTracker {
    void NotifyTerminated(ServiceProcessId id);
  
--  void NotifyCrashed(ServiceProcessId id);
-+  void NotifyCrashed(ServiceProcessId id, int exit_code);
+   void NotifyCrashed(ServiceProcessId id,
+-                     UtilityProcessHost::Client::CrashType type);
++                     UtilityProcessHost::Client::CrashType type,
```

Comment:
Oh `UtilityProcessHost::Client::CrashType` is interesting to expose for utility process api